### ISSUE TITLE
All FIELDSET tags should be labeled with LEGEND tags

### ISSUE THREAD

**Issue by hortongn** [uclibs/scholar_uc#514]
The first element inside the FIELDSET must be a LEGEND element, which provides a label or description for the group.		

Problem pages:
* https://scholar-qa.uc.edu/catalog
* https://scholar-qa.uc.edu/catalog?f%5Bgeneric_type_sim%5D%5B%5D=Collection
* https://scholar-qa.uc.edu/catalog?f%5Bgeneric_type_sim%5D%5B%5D=Work
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Article
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Collection	
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Document
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Image
* https://scholar-qa.uc.edu/hydramata/groups/new
* https://scholar-qa.uc.edu/works/articles/new	
* https://scholar-qa.uc.edu/works/datasets/new	
* https://scholar-qa.uc.edu/works/documents/new	
* https://scholar-qa.uc.edu/works/generic_works/new	
* https://scholar-qa.uc.edu/works/images/new
